siteimprove/admin/js/siteimprove.js

```diff
@@ -121,6 +121,11 @@
       if( this.method === "domain" ){
         _si.push(['input', this.url, this.token, function() { console.log('Inputted new javascript overlay file'); } ]); 
       } else {
+        // 0 = overlay-latest.js
+        // 1 = overlay-v1.js
+        if(this.version == 0 && this.preview) {
+          _si.push(['registerPrepublishCallback', getDomCallback, this.token]);
+        }
         _si.push([this.method, this.url, this.token]);
       }
```

Comment:
Not exactly to this line, but a few lines down from here: In the call to "clear" if the regex pattern is matching, we do a `console.log` . Should we really be doing this? This seems like some debugging code that has somehow made it into the code, but shouldn't be there.

(I've been comparing this branch with `master` to see what essentially would be the difference to the current released version, which is how I found this).